.github/workflows/ci_api.yml

```diff
@@ -6,6 +6,7 @@ on:
     types: [opened, synchronize, reopened, edited]
     paths:
       - "api/**"
+  workflow_dispatch:
```

Comment:
This will allow us to API re-deployment manually from GitHub in case we ever need to, which I think would be handy (e.g. if someone forgets to / wrongly sets the environment variables for the API)

Reply:
I had no idea how to add that, every day is a school day!